src/lib/registration/connect_helpers.rb

```diff
@@ -142,6 +142,7 @@ def self.catch_registration_errors(message_prefix: "",
         # update the message when an old SMT server is found
         check_smt_api(e.message)
         details_error(message_prefix + _("Cannot parse the data from server."), e.message)
```

Comment:
while Yast::Report returned `nil`, details_error reports `:ok` so, added the return value explicitly as we have in the other rescued exceptions